components/Starknet/modules/quick-start/pages/environment-setup.adoc

```diff
@@ -1 +1,152 @@
-include::1.0.0@docs-common-content:ROOT:partial$partial_environment_setup.adoc[]
+[id="setup"]
+= Setting up your environment
+
+In order to interact with Starknet and compile Cairo code, you need to install several tools.
+
+The following tools are recommended to begin developing on Starknet:
+
+[cols="1,1,1,1"]
+[%autowidth.stretch]
+|===
+|Tool name | Description | Documentation |Code Repository
+
+|Starkli
+|A command-line interface that allows you to interact with Starknet.
+|https://book.starkli.rs/[book.starkli.rs]
+|https://github.com/xJonathanLEI/starkli[github.com/xJonathanLEI/starkli]
+
+|Scarb
+|A build toolchain and package manager for Cairo and Starknet ecosystems.
+|https://docs.swmansion.com/scarb/[docs.swmansion.com/scarb]
+|https://github.com/software-mansion/scarb[github.com/software-mansion/scarb]
+
+|===
+
+[#installing_starkli]
+== Installing Starkli
+
+The steps for installing Starkli and upgrading Starkli are identical.
+
+.Procedure
+
+. Install Starkliup, the installer for the Starkli environment:
++
+[source,shell]
+----
+curl https://get.starkli.sh | sh
+----
++
+Starkliup should now be installed.
+
+. Restart the terminal.
+
+. Install Starkli:
++
+[source,shell]
+----
+starkliup
+----
++
+Starkli should now be installed.
+
+. Restart the terminal and run the following command to verify the installation:
++
+[source,shell]
+----
+starkli --version
+----
++
+Starkli prints the current version.
+
+[#setting_environment_variables_for_starkli]
+== Setting environment variables for Starkli
+
+For the majority of flags available on Starkli you can set environment variables to make the commands shorter and easier to manage.
+
+Setting environment variables for Starkli significantly simplifies command execution and management, thereby enhancing efficiency, readability, and control when using Starkli.
+
+The two primary environment variables that are vital for effective usage of Starkli's CLI are:
+
+[horizontal,labelwidth="25"]
+`STARKNET_ACCOUNT`:: The location of the xref:set-up-an-account.adoc#creating_an_account_descriptor[Account Descriptor] file.
+`STARKNET_KEYSTORE`:: The location of the keystore file for the xref:set-up-an-account.adoc#creating_a_signer[Signer].
+
+Set these environment variables as follows:
+
+[source,bash]
+----
+export STARKNET_ACCOUNT=~/.starkli-wallets/deployer/account.json
+export STARKNET_KEYSTORE=~/.starkli-wallets/deployer/keystore.json
+----
+
+[#installing_scarb]
+== Installing Scarb
+
+Scarb is compatible with macOS, Linux, and Windows operating systems.
+
+
+=== MacOS and Linux installation
+
+It is highly recommended to install Scarb via the link:https://asdf-vm.com/[asdf] version manager.
+
+Installing Scarb with `asdf` seamlessly integrates with the Cairo VSCode extension, and allows to easily switch between different versions of Scarb, globally or per project.
```

Comment:
Please be consistent and use either asfd or `asfs` throughout the entire subsection